static/css/awards.css

```diff
@@ -0,0 +1,64 @@
+#awards {
+   
+    
+}
+ #awards .container-fluid {
+margin : auto;
+}
+#awards .card {
+    border : 0.1px solid rgb(180, 176, 176);
+    border-radius : 2%;
+    height:320px;
+    width : 300px;
+    display: inline-block;
+    margin: 20px;
+vertical-align: top;
+box-shadow: 2px 2px 2px 2px rgb(185, 184, 184);
+-webkit-transition: box-shadow 0.3s ease-out;
+  -moz-transition: box-shadow 0.3s ease-out;
+  -o-transition: box-shadow 0.3s ease-out;
+  transition: box-shadow 0.3s ease-out;
+}
+
+#awards .card:hover {
+    box-shadow: 0.8px 0.8px 0.8px 0.8px rgb(185, 184, 184);
+}
+
+
+
+#awards .row{
+    padding: 20px;
+    text-align: center;
+}
+#awards .logo {
+    /* border : 1px solid rgb(180, 176, 176); */
+     height: 150px;
+     margin: 10px;
+    
+  
+}
+#awards .logo img {
+   width : 130px;
+   height: 130px;
```

Comment:
you are fixing the both height & width of the image, that's why they are stretched for non square logos.

Reply:
I did this because when i add logos some are big and some are small. I wanted a fixed size. Is any of the logos looking weird ?.I thought the square logos would look good in this section that's why i chose this.
If not, then can how will I make all the logos of the same size  ?  
@CosmicCoder96